core/templates/domain/skill/SkillDifficultyObjectFactorySpec.ts

```diff
@@ -45,5 +45,25 @@ describe('Skill Difficulty object factory', () => {
       };
       expect(skillDifficulty.toBackendDict()).toEqual(skillDifficultyDict);
     });
+
+    it('should set the description of the skill difficulty instance', () => {
+      var skillDifficulty = skillDifficultyObjectFactory.create(
+        '1', 'test skill', 0.3);
+      expect(skillDifficulty.getId()).toEqual('1');
+      expect(skillDifficulty.getDescription()).toEqual('test skill');
+      expect(skillDifficulty.getDifficulty()).toEqual(0.3);
+      skillDifficulty.setDescription('new test skill');
+      expect(skillDifficulty.getDescription()).toEqual('new test skill');
+    });
+
+    it('should set the difficulty of the skill difficulty instance', () => {
+      var skillDifficulty =
+        skillDifficultyObjectFactory.create('1', 'test skill', 0.3);
```

Comment:
Can we do similar to line 50 here i.e, break line after `(`?

Reply:
Done.